### ISSUE TITLE
Timezone issues

### ISSUE THREAD

**Issue by DannyBen** [hellogerard/jobby#41]
It seems that jobby runs at EST timezone on my machine.

The server is configured to show a different timezone, and I also tried using `date_default_timezone_set` but the cron expressions are still considered as if they are EST.

Is this by design, or a bug, or have I missed something?

**Comment by hellogerard** [comment]
I don't believe this has anything to do with jobby. Jobby doesn't do any timezone-related logic. I believe that this means that it simply uses the timezone of the server (i.e. the time that you see when you run `date` in the shell, not the timezone of PHP).

Usually, this is okay since the times created by your app runs on the same server and thus uses the same timezone. If your app user's enter local time in a different timezone, and then you have a jobby job operating on those times in EST, then it's up to the app to make that translation.

**Comment by DannyBen** [comment]
I am not sure what is the reason, but it doesnt run at the expected time now.
And, I do believe Jobby should obey PHP timezone, since it is a PHP script...

**Comment by parkerj** [comment]
Commenting since this is still open. A good reason for this happening could be that timezone is not set in php.ini. Set it in php.ini restart sever if necessary, and see if you still get the same error.

**Comment by hellogerard** [comment]
> Commenting since this is still open. A good reason for this happening could be that timezone is not set in php.ini. Set it in php.ini restart sever if necessary, and see if you still get the same error.

Does this work?

**Comment by Axierty** [comment]
If i set timezone in php.ini , the timezone is right, but i set it in script , i still show the EST timezone
